### ISSUE TITLE
Cant GET an event-source-mapping

### ISSUE THREAD

**Issue by aventurella** [localstack/localstack#648]
Similar to #343, but for `GET`

```
 aws lambda list-event-source-mappings --endpoint http://localhost:4574
```
Properly provides a JSON response:

```json
{
    "EventSourceMappings": [
        {
            "UUID": "534087f0-27ca-4a96-81ee-32006bd91900",
            "BatchSize": 100,
            "EventSourceArn": "arn:aws:kinesis:us-east-1:000000000000:stream/my-stream",
            "FunctionArn": "arn:aws:lambda:us-east-1:000000000000:function:my_function",
            "LastModified": 1520468565.0,
            "LastProcessingResult": "OK",
            "State": "Enabled",
            "StateTransitionReason": "User action"
        }
    ]
}

```

However when trying to:
 
```
aws lambda get-event-source-mapping --uuid 534087f0-27ca-4a96-81ee-32006bd91900  --endpoint http://localhost:4574
```

```
An error occurred (405) when calling the GetEventSourceMapping operation: <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>405 Method Not Allowed</title>
<h1>Method Not Allowed</h1>
<p>The method is not allowed for the requested URL.</p>
```

It's making this request:

```
aws lambda get-event-source-mapping --uuid 534087f0-27ca-4a96-81ee-32006bd91900  --endpoint http://localhost:4574 --debug

 DEBUG - Making request for OperationModel(name=GetEventSourceMapping) (verify_ssl=True) with params: {'url_path': '/2015-03-31/event-source-mappings/534087f0-27ca-4a96-81ee-32006bd91900', 'query_string': {}, 'method': 'GET', 'headers': {'User-Agent': 'aws-cli/1.14.30 Python/3.6.4 Darwin/17.4.0 botocore/1.8.34'}, 'body': b'', 'url': 'http://localhost:4574/2015-03-31/event-source-mappings/534087f0-27ca-4a96-81ee-32006bd91900', 'context': {'client_region': 'us-west-2', 'client_config': <botocore.config.Config object at 0x10ee1ccc0>, 'has_streaming_input': False, 'auth_type': None}}
```

**Comment by aventurella** [comment]
Ah...
https://github.com/localstack/localstack/blob/master/localstack/services/awslambda/lambda_api.py#L673

`get_event_source_mapping`  is missing would seem to be the cause. I might try my hand at PRing this.

**Comment by aventurella** [comment]
Ok, so this is all that needs to be added to:

https://github.com/localstack/localstack/blob/master/localstack/services/awslambda/lambda_api.py

```python
@app.route('%s/event-source-mappings/<mapping_uuid>' % PATH_ROOT, methods=['GET'])
def get_event_source_mapping(mapping_uuid):
    """ Get an existing event source mapping
        ---
        operationId: 'getEventSourceMapping'
        parameters:
            - name: 'request'
              in: body
    """
    mappings = event_source_mappings
    mappings = [m for m in mappings if mapping_uuid == m.get('UUID')]

    if len(mappings) == 0:
        return error_response('The resource you requested does not exist.', 404, error_type='ResourceNotFoundException')
    return jsonify(mappings[0])
```

Now to add the unit test

**Comment by whummer** [comment]
Fixed in #652 - thanks!
